apps/admin/src/components/prompts/standard/submit-prompt.vue

```diff
@@ -1,16 +1,41 @@
 <template>
-  <v-tab-item key="options" value="options"></v-tab-item>
+  <v-tab-item key="options" value="options">
+    <v-card-title>{{ $t('survey-schemes.prompts.submit-prompt.optionTitle') }}</v-card-title>
```

Comment:
rename the key to something like `review` ? ...There will be more an more options once more features implemented. Maybe one title is enough, see options in other prompt.